### PR TITLE
Support/tvos

### LINKED ISSUE #176: Possible support for tvOS
Hi,

is it possible to support tvOS without any development? i want to use modernavplayer with my tvOS target but pod install says its not compatible with tvOS

### PR DESCRIPTION
**Pull request by yaroslavlvov**
Add tvOS support

tvOS example screenshot
image

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 
 ---
+## [1.7.2]
```
Comment (raphrel) [inline CHANGELOG.md]: should be [X.X.X] - version depends of new features added

**Review by raphrel — CHANGES_REQUESTED** [review]
I really appreciate your PR, thanks !

Before accepting, we have to make sure it's working with a demo. 
could you add a tvOS demo target in the project with a simple example like in iOS project ?

Also, please add tvOS 10.0 requirements in README file 🙏

File: CHANGELOG.md (hunk 1 above)
Comment (yaroslavlvov) [reply CHANGELOG.md]: Do you mean literally [X.X.X]?

File: README.md [hunk 2]
```diff
@@ -30,6 +30,7 @@
 ## Requirements
 
 - iOS 10.0+
+- tvOS 10.0+
```
Comment (yaroslavlvov) [inline README.md:33]: here is a tvOS in requirements, what else would you like to be added in README.md?

File: CHANGELOG.md (hunk 1 above)
Comment (raphrel) [reply CHANGELOG.md]: yep :) `## [X.X.X]`

Comment (yaroslavlvov) [reply CHANGELOG.md]: changed

File: Podfile [hunk 3]
```diff
@@ -14,3 +14,8 @@ target 'ModernAVPlayer_Example' do
 
   end
 end
+
+target 'ModernAVPlayer_Example_tvOS' do
```
Comment (raphrel) [inline Podfile:18]: I still have some issues when fetch your pr and build.

When executing pod install, I see some new files added and everything's fine. 
can you try again from your side a new pod install and let me know ?

Screenshot 2020-11-18 at 12 59 37

**Review by raphrel — CHANGES_REQUESTED** [review]

File: Podfile (hunk 3 above)
Comment (yaroslavlvov) [reply Podfile:18]: Included those changed into PR with a latest commit

**Review by raphrel — APPROVED** [review]

**Comment by raphrel** [comment]
thanks again !!

I may release a new version tonight

**Pull request merged**